### DISCUSSION TITLE
A Miryoku inspired journey

### DISCUSSION THREAD

**Discussion by brickbots** [manna-harbour/miryoku#60] [Show and tell]
I'm not sure how most people end up using a 36(ish) key layout for their daily driver, but for me it started with a Planck keyboard that a co-worker gave me.  I learned the standard keymap that comes with the board, and used it a for a while, but the physical layout just felt a bit cramped... so I bought a levinson (let's split) kit so I could split my hands a bit.

This was more comfortable, but the keymap was still a bit of a mess and the experience was still not as awesome as I thought it could be.  The Kyria caught my eye and I got my hands on a kit which I built up.  It felt much better, and I started customizing my keymaps via QMK, which helped a bit... but now I had multiple boards with different physical AND logical layouts.  

I ended up building another Kyria kit just so my work and home computer could have identical layouts, but I was still also using the levinson from time to time on another computer and my WPM speed on all the boards was pretty poor due to all the switching.  It was about this time that I bumped into the Miryoku layout... I wish I remembered exact what reddit post showed me the way, but it was indeed a revelation!

The layout made a huge amount of sense, and was obviously more thought out than the one I'd been working on improving for the Kyria.  More critically, the awesome ecosystem that allows it to be ported and used on such a wide variety of boards meant that I could use the same logical layout across all my boards.  It was like manna from heaven, as the expression goes.

The only downside... I had ALL these extra keys on my board.  Look at them all...just taking up space and doing nothing at all.  So I printed a specific case for my Levinson to block out those pesky extra keys:
![miryoku_levinson](https://user-images.githubusercontent.com/11450005/147172022-089ef61f-d9e8-4c50-a70a-3acf83d89a11.png)

That worked great, but then I saw the FiFi in one of Manna Harbour's Miryoku images!  It's the Kyria column layout, without all those pesky extra keys!  I fell hard.....
![fifi](https://user-images.githubusercontent.com/11450005/147172124-11f0bf0c-00bc-4acd-8379-9fbe30764a6e.jpeg)

When I built a cyberdeck... it had to be Miryoku!  This saved a bunch of space actually and let me fit a full keyboard in a small space:
![cyberdeck](https://user-images.githubusercontent.com/11450005/147172245-e5db3503-e181-4f85-ac0d-c745b7dcf684.jpeg)

And then had to try wireless!  Which was a breeze as Miryoku had already been partially ported to ZMK 👍 
![sci_fifi](https://user-images.githubusercontent.com/11450005/147172142-7968ce04-4dd7-49ac-b524-ffec0af330e3.jpg)

Recently I was motivated to learn PCB design, at least enough for a keyboard, by the Miryoku layout and the BabyV (which has a few too many keys, but its pretty awesome otherwise)
![family](https://user-images.githubusercontent.com/11450005/147172373-02dc9910-40df-4267-9e62-bd1ffbab4d5e.jpeg)

These have all been incredibly fun projects and there is NO way I would have embarked on them if I had to think about what layout to use for each.  Having a flexible, consistent, minimal layout has enabled this fun exploration and I'm very grateful to our host here for the efforts they have put forth to develop this layout and the amazing amount of work to make it compatible with so many boards across three different implementation platforms.  

Thanks for coming along on this rambling journey and thank you Manna Harbour 🎉

**Comment by manna-harbour** [comment]
Thanks so much for sharing your journey!  It's been a real pleasure following along with you from the beginning.  I'm looking forward to your future creations!
